dask_kubernetes/operator/controller/tests/test_controller.py

```diff
@@ -193,7 +193,7 @@ async def test_simplecluster(k8s_cluster, kopf_runner, gen_cluster):
                     "jsonpath='{.items[0].metadata.annotations}'",
                 )[1:-1]
             )  # First and last char is a quote
-            assert service_annotations == _EXPECTED_ANNOTATIONS
+            assert _EXPECTED_ANNOTATIONS <= service_annotations
```

Comment:
Adding a handler to the service broke this test. Tweaking it to check that `_EXPECTED_ANNOTATIONS` are a subset of the annotations to avoid this kind of break in the future. cc @bstadlbauer